en/user_guide/content.rst

```diff
@@ -0,0 +1,58 @@
+Contents
+========
+
+Contents are managed in the Back Office.
+
+
+Listing
+-------
+Contents listing is accessed in the Editorial section, by clicking on the "Contents" link. A submenu
+lists all Content Types availables.
+
+.. image:: ../../images/content_list_submenu.png
+
+By clicking a Content Type, the list of related contents appears.
+
+.. image:: ../../images/content_list.png
+
+This list has a pagination and filters, and shows the name, the status, the version and the language
+of existing contents. From this list, new content can be created, and existings ones can be edited or
+deleted.
+
+
+Creation
+--------
+From the content list, creation form can be access by clicking on the "Add" button. Informations asked
+depend on the Content Type, each Content Type has its own form. By clicking on the "Save" button, the
```

Comment:
depend on the Content Type, each Content -> depend on the Content Type: each Content